vmc/resource_vmc_cluster.go

```diff
@@ -24,7 +24,16 @@ func resourceCluster() *schema.Resource {
 		Update: resourceClusterUpdate,
 		Read:   resourceClusterRead,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ",")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" || !IsValidUUID(idParts[0]) || !IsValidUUID(idParts[1]) {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected id,sddc_id", d.Id())
```

Comment:
Since we are enforcing len(idParts) should be 2, isn't the empty check redundant for idParts[0] and idParts[1]? If they are empty the split on line 28 should return shorter length array..